DL_VerticalRequest.cpp

```diff
@@ -0,0 +1,143 @@
+#include <iostream>
```

Comment:
please remove the .exe  pushed by mistake

Reply:
i remove .exe file published by mistake